### DISCUSSION TITLE
Add `luau` as a supported programming language

### DISCUSSION THREAD

**Discussion by Epikest** [community/community#19773] [Other Feature Feedback, Questions, & Ideas]
currently, GitHub does not recognize any `luau` files. this leads to issues such as:
- no syntax highlighting
- no codebase composition
![image](https://user-images.githubusercontent.com/45499416/176744669-fbf3cfa8-c766-489a-b7f3-083ffc0ae843.png)

**Comment by Jack5079** [comment]
GitHub language support is handled by Linguist: https://github.com/github/linguist/issues/new?assignees=&labels=Add+Language%2C+Good+First+Issue&template=new_language.md&title=

**Reply by phoriah** [reply]
Already Exists: https://github.com/github-linguist/linguist/issues/6000
Go Support it 😁
